platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/loader/LibraryLoaderProviderImpl.java

```diff
@@ -23,36 +24,25 @@
    */
   @Override
   public LibraryLoader getDefaultLibraryLoader() {
-    return new ReLinkerLibraryLoader();
+    return new SoLibraryLoader();
   }
 
   /**
    * Concrete implementation of a LibraryLoader using ReLinker.
    */
-  private static class ReLinkerLibraryLoader extends LibraryLoader {
+  private static class SoLibraryLoader extends LibraryLoader {
+
+    private static final String TAG = "SoLibraryLoader";
 
     @Override
     public void load(String name) {
       try {
-        Context context = Mapbox.getApplicationContext();
-        ReLinker.log(new LibraryLogger()).loadLibrary(context, name);
+        init(Mapbox.getApplicationContext(),false);
```

Comment:
What does `false` mean in this context? could its value bring any regressions? I'd add a comment or add a named variable.

Reply:
this references to exoPackage which is used when building with [Buck](https://buck.build/article/exopackage.html).
Added a small comment with a link to exopackage.